### PR TITLE
Update rxdart to 0.23 and run flutter format to improve package health

### PR DESCRIPTION
**Pull request by rafaelcmm**
## Description

*Dart on 2.6 powered Stream object, deprecating observable. All observable features were incorporated into dart Stream*

*This PR has a goal to improve health of flutter_clean_architecture package and bring some performance improvements with the rxdart 0.23 update and dart update to 2.6.*

*Also, I perform flutter format in the entire project to improve pub package health* 

## Developer checklist

- [X] All steps on circleci are passing.

## For reviewers

All PR's needs at least one reviewer to be merged.

Before merge this PR confirm that it meets all requirements listed below:

- The PR has good quality code and has tests (if is the case)
- This PR is not checked with WIP on title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -345,7 +345,7 @@ class LoginUseCase extends CompletableUseCase<LoginUseCaseParams> {
   @override
   // Since the parameter type is void, `_` ignores the parameter. Change according to the type
   // used in the template.
-  Future<Observable<void>> buildUseCaseObservable(params) async {
+  Future<Observable<void>> buildUseCaseStream(params) async {
```
Comment (ShadyBoukhary) [inline README.md]: Please also update the docs, including the README.md

File: lib/src/usecase.dart [hunk 2]
```diff
@@ -39,7 +39,7 @@ import 'dart:async';
 ///     @override
 ///     // Since the parameter type is void, `_` ignores the parameter. Change according to the type
 ///     // used in the template.
-///     Future<Observable<GetSponsorsUseCaseResponse>> buildUseCaseObservable(_) async {
+///     Future<Observable<GetSponsorsUseCaseResponse>> buildUseCaseStream(_) async {
```
Comment (ShadyBoukhary) [inline lib/src/usecase.dart]: Change to stream

File: lib/src/usecase.dart [hunk 3]
```diff
@@ -147,7 +147,7 @@ abstract class UseCase<T, Params> {
 ///       LogoutUseCase(this._authenticationRepository);
 ///
 ///       @override
-///       Future<Observable<User>> buildUseCaseObservable(void ignore) async {
+///       Future<Observable<User>> buildUseCaseStream(void ignore) async {
```
Comment (ShadyBoukhary) [inline lib/src/usecase.dart]: Change to stream

**Comment by ShadyBoukhary** [comment]
I will also merge this into development instead because these breaking changes will require a new release. Once it is in development I will update the version then push it to master.

**Comment by rafaelcmm** [comment]
@ShadyBoukhary sorry about that, I forgot some docs. 

Already fixed it.

**Comment by rafaelcmm** [comment]
> I will also merge this into development instead because these breaking changes will require a new release. Once it is in development I will update the version then push it to master.

@ShadyBoukhary sorry about that, I forgot some docs.

Already fixed it. You can merge it to development and generate the version and update the changelogs later

**Comment by ShadyBoukhary** [comment]
#23

**Pull request merged**